### PR TITLE
Add --default-filters parameter

### PR DESCRIPTION
**Pull request by tygill**
This change adds the --default-filters parameter, which adds built-in blacklists for common files / folders that don't need to be backed up. This includes things like temp files/folders, caches for common programs, and operating system files.

These filters are grouped into three sets, one each for Windows, Linux, and OSX. (When --default-filters is used without a value, it will automatically select the filters for the current operating system, or the sets to use can be added explicitly.)

The contents for these filters come from this question, though some have been adapted or removed:
https://superuser.com/questions/443890/what-files-file-types-and-folders-to-exclude-from-user-folder-backup

This change adds help / documentation information, and adds these filters to the web UI.

### CODE REVIEW

File: Duplicati/CommandLine/Help.cs [hunk 1]
```diff
@@ -84,6 +84,8 @@ public static void PrintUsage(TextWriter outwriter, string topic, IDictionary<st
                     topic = "exclude";
                 else if (System.Environment.CommandLine.IndexOf("--include", StringComparison.InvariantCultureIgnoreCase) >= 0)
                     topic = "include";
+                else if (System.Environment.CommandLine.IndexOf("--default-filters", StringComparison.InvariantCultureIgnoreCase) >= 0)
```
Comment (kenkendk) [inline Duplicati/CommandLine/Help.cs]: I don't think this is required as the help routine will look at options if no match is found in the help file.

**Comment by kenkendk** [comment]
Awesome work!

I will merge "as-is" because this is already a huge help.
I think we should add Duplicati's own folders also, as mentioned in #1443.

**Comment by MM-John** [comment]
Looks like there's a bug in the list in [superuser](https://superuser.com/questions/443890/what-files-file-types-and-folders-to-exclude-from-user-folder-backup), repeated here. Under All, this entry `.*/(cookies|permissions).sqllite(-.{3})?` has two `l`'s (lower case `L`'s) in `.sqlite`

**Comment by MM-John** [comment]
Yeah. I was looking in commit:
https://github.com/duplicati/duplicati/pull/2740/commits/d4865eb4cdb564399f8234944a5e33480cd4653e
`/Duplicati/Library/Utility/DefaultFilters.cs`
aka:
https://github.com/tygill/duplicati/blob/d4865eb4cdb564399f8234944a5e33480cd4653e/Duplicati/Library/Utility/DefaultFilters.cs

But it looks like `DefaultFilters.cs` isn't even there anymore.

Apparently I just clicked on:
image
above and didn't really know what I was doing :P

**Pull request merged**